firmware/src/transceiver.c

```diff
@@ -545,9 +557,12 @@ static inline void PrepareRDMResponse() {
   if (g_timing_settings.rdm_responder_jitter) {
     jitter = Random_PseudoGet() % g_timing_settings.rdm_responder_jitter;
   }
+  // It's important to stop the timer before changing the period, see 14.3.11
```

Comment:
Is it worth a regex in our cpplint task or similar that ensures PLIB_TMR_Period16BitSet and others are always preceded by PLIB_TMR_Stop?